app/Models/RequestPurchaseOrder.php

```diff
@@ -41,4 +74,55 @@ public function requestCanvassSummary()
     {
         return $this->belongsTo(RequestCanvassSummary::class, 'request_canvass_summary_id');
     }
+
+    /**
+     * ==================================================
+     * MODEL ATTRIBUTES
+     * ==================================================
+     */
+
+    public function getIsPrepaymentAttribute(): bool
+    {
+        return $this->requestCanvassSummary
+            ? strtolower($this->requestCanvassSummary->terms_of_payment) === 'prepayment in full'
```

Comment:
gamiti palihog og enum sa terms_of_payment if naka setup ta og enum ani mark. pero if naka string ra ni. ayaw i disregard ra ni na comment. pa reply lang if string ra ni nga column and wala tay naka setu p nga enums aning terms_of_payment mark